Assets/Mirror/Tests/Editor/NetworkManagerStopHostOnServerDisconnectTest.cs

```diff
@@ -27,13 +27,13 @@ public override void SetUp()
 
         // test to prevent https://github.com/vis2k/Mirror/issues/1515
         [Test]
-        public void StopHostCallsOnServerDisconnectForHostClient()
+        public void StopClientCallsOnServerDisconnectForHostClient()
```

Comment:
why remove the old test?
shouldn't we keep it and duplicate for a new one?

Reply:
Added test and another change to network manager